fastchat/model/model_adapter.py

```diff
@@ -338,6 +362,14 @@ def load_model(
 
     if device == "xpu":
         model = torch.xpu.optimize(model, dtype=kwargs["torch_dtype"], inplace=True)
+    else:
+        try:
+            from optimum.bettertransformer import BetterTransformer
```

Comment:
Can we make it an optional transformation instead of a default behavior?

Reply:
I think it's possible. I need to check it :)
If it is verry opnional, I appologies by advance to added it :/

Reply:
Using of optimum.bettertransformer is not mandatory. It's in a try / catch to use the package only if it's installed. You think it's will be interesting to have to install the package and pass a parameter to use the optimum library ?

Reply:
@DevNullx64  The current code still runs optimum by default, although there is a try/catch to handle the errors.
This can cause some unexpected behaviors. Could you update your code, so that optimum is only used when a user explicitly specifies it?

Reply:
'Try to do it this week end ;)